docker/prod/nginx.conf

```diff
@@ -53,13 +53,7 @@ http {
         gunzip on;
         gzip_static on;
 
-        location /oauth2/tokens/authService {
-            rewrite  ^/oauth2/tokens/authService /oauth2/tokens break;
-            proxy_pass       $AUTHSERVICE_CORTEX_URL;
-        }
-
-        location ~ ^/cortex/[0-9a-z]+/authService {
-            rewrite  ^/authService / break;
+        location /admin {
             proxy_pass       $AUTHSERVICE_CORTEX_URL;
```

Comment:
```suggestion
            proxy_pass       $AUTHSERVICE_URL;
```

Reply:
@deepakazad Thanks! I agree. I'll make this update.